### PR TITLE
Move the initial setup to a separate script

### PR DESCRIPTION
**Pull request by Erisa**

### CODE REVIEW

File: modules/setup.rb [hunk 1]
```diff
@@ -9,67 +9,8 @@ class << self
...
-    newconfig['master_owner'] = id
-
-    File.write('config/config.yml', newconfig.to_yaml)
-
-    puts 'All done! The bot will launch in 1 second..'
-    # Delay so the user knows what's happening.
-    sleep(1)
-
-    # Let's go!
+    puts 'No valid config file found. Please copy config/config.sample.yml
```
Comment (spotlightishere) [inline modules/setup.rb:12]: We should probably exit here, so that there aren't odd errors about missing/nil so forth afterwards.

Comment (Erisa) [reply modules/setup.rb:12]: I had considered this, the resulting traceback says, among other things, `No such file or directory @ rb_sysopen - config/config.yml` which I thought was an appropriate reason to exit, can change to cleanly exit if you disagree though.

Comment (Erisa) [reply modules/setup.rb:12]:
```
erisa@Tuturu /m/c/U/E/O/W/B/Yuuki-Bot (master)> bash run_linux.sh
No valid config file found. Please copy config/config.sample.yml to config/config.yml and edit it, or run config.rb.
Traceback (most recent call last):
        7: from run.rb:3:in `<main>'
        6: from run.rb:30:in `<module:YuukiBot>'
        5: from run.rb:30:in `require_relative'
        4: from /mnt/c/Users/Erisa/OneDrive/Work/Bot/Yuuki-Bot/modules/setup.rb:3:in `<top (required)>'
        3: from /mnt/c/Users/Erisa/OneDrive/Work/Bot/Yuuki-Bot/modules/setup.rb:15:in `<module:YuukiBot>'
        2: from /usr/lib/ruby/2.7.0/psych.rb:577:in `load_file'
        1: from /usr/lib/ruby/2.7.0/psych.rb:577:in `open'
/usr/lib/ruby/2.7.0/psych.rb:577:in `initialize': No such file or directory @ rb_sysopen - config/config.yml (Errno::ENOENT)```

Comment (spotlightishere) [reply modules/setup.rb:12]: Ah alright, sounds good - I didn't actually give this a run. After more thorough testing, looks good to me.

**Pull request merged**